### PR TITLE
Mphys helper functions

### PR DESCRIPTION
**Pull request by A-CGray**
Adds a helper function to the TACS MPhys wrapper for adding constraints. `add_tacs_constraints` takes in an OpenMDAO model and an MPhys scenario and automatically adds any TACS constraints it finds in the scenario as constraints in the model. Saves the user having to manually call `add_constraint` for all of their TACS constraints. 

I put it in a `utils` file for now, happy to take suggestions on a better place for it.

### CODE REVIEW

**Comment by timryanb** [comment]
@A-CGray, the log function call looks helpful, but it seems more like a general OpenMDAO feature than specific to TACS. Could we reach out to OpenMDAO/OpenMDAO and see if they'd be willing to adopt this feature (or such a feature already exist)?

**Comment by A-CGray** [comment]
Sure, I'll remove it from this PR and bring it up with the OpenMDAO team

File: tacs/mphys/utils.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    function yourself.
+
+    This function relies on you having defined the lower and/or upper bounds
+    of the constraints when you created them in your `constraint_setup`
+    function.
+
+    Parameters
+    ----------
+    model : OpenMDAO model
+        Model to add constraints to
```
Comment (timryanb) [inline tacs/mphys/utils.py:18]: Is the `model` really needed as an input here? It looks like it's only used to get the root proc for printing? Can we just use the comm from the scenario?

Comment (A-CGray) [reply tacs/mphys/utils.py:18]: You need the model because that's what the constraints are added to

Comment (A-CGray) [reply tacs/mphys/utils.py:18]: Oh, can call `add_constraint` on a scenario too? In that case I agree and will remove the model input

Comment (timryanb) [reply tacs/mphys/utils.py:18]: Ah, I see that now. But you can just add the constraints directly to the subsystem and skip the model. Something like: `constraint.add_constraint(name, equals=lb, linear=True)`

Comment (timryanb) [reply tacs/mphys/utils.py]: Ah, nvm. Beat me to it

**Review by timryanb — APPROVED** [review]

**Comment by A-CGray** [comment]
@timryanb looks there is an OpenMDAO bug that results in an indexing error if you only have linear constraints, I'll open an issue on the OpenMDAO repo

**Pull request merged**